packages/cloudflare/src/utils/image-config.ts

```diff
@@ -4,7 +4,7 @@ import { passthroughImageService, sharpImageService } from 'astro/config';
 export function setImageConfig(
 	service: string,
 	config: AstroConfig['image'],
-	command: 'dev' | 'build' | 'preview',
+	command: 'dev' | 'build' | 'preview' | 'sync',
```

Comment:
I think it would be better to use `HookParameters<'astro:config:setup'>['command']`